targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala

```diff
@@ -86,11 +103,61 @@ object TargetBody {
                 update.runS(input).value
             }
           ) _
+
           val modifyName = undoSet[NonEmptyString](
             unsafeTargetName,
             n => EditSiderealInput.name.set(n.value.some)
           ) _
 
+          val modifyEpoch = undoSet[Epoch](
```

Comment:
@rpiaggio There is some amount of repetition here. any ideas on how to reduce it?

Reply:
Although they follow a similar pattern, I can't think of a way of reusing things: each invocation to `undoSet` needs its own `Lens` and `EditSiderealInput`.